### DISCUSSION TITLE
Users can't login to Wiki.js

### DISCUSSION THREAD

**Discussion by Kimyeti** [requarks/wiki#2925] [Help / Questions] [answered]
Hi

I can't of the life of me get the LDAP authentication to work. To isolate the issue I've created a brand new Windows AD server and Wiki.js instance to test the ldap config between the servers. 

Wiki.js config:

![merged](https://user-images.githubusercontent.com/10739330/104726859-ab5f5680-5734-11eb-8301-f628bd439aac.jpg)

Windows AD server:

![ad-server](https://user-images.githubusercontent.com/10739330/104726904-bdd99000-5734-11eb-95eb-11028e57a66d.png)

Trying to login:

![2021-01-15 13_01_52-Window](https://user-images.githubusercontent.com/10739330/104726968-da75c800-5734-11eb-9002-b6a9c7af6814.png)

"nel" is the user Neville Longbottom who is in the same user folder as the administrator on the AD server. I can login with the "nel" initials on a regular Windows PC.

Does anybody know what I'm missing here? I'm happy to provide more screenshot and details. Thank you in advance.

**Comment by Just-Insane** [comment, marked as answer]
Try setting your Search Base to `CN=Users,DC=yeti,DC=local`

Also might need to change the username fields to `(sAMAccountName={{username}})` instead of `(uid={{username}})`.

im assuming you can’t login as “Administrator” right now either?

**Reply by Kimyeti** [reply]
Thank you the issue has been resolved.

I'll provide a screenshot here of the final configuration to help others that encounters the same problem.

![merged2](https://user-images.githubusercontent.com/10739330/105047507-b5ea5a80-5a6a-11eb-86b4-0b1f7dd8a16f.jpg)

**Reply by s3rvant** [reply]
Thank you for posting your solution; it worked for me as well. Enabling LDAP flag didn't work (no log entries generated) so had hit a wall.

**Reply by DrySock** [reply]
Thanks a lot, helped for me

**Reply by higorblands** [reply]
Thanks, it solved my problem too :D

**Reply by Erick86** [reply]
I have the same issue but I'm unable to login to wiki.js through active directory. 

ldap url: ldap://10.250.10.15:389
Admin Bind DN: CN=Administrator,CN=Users,DC=test,DC=local
Admin Bind Credentials: Mypassword123!
Search Base: OU=TEST,dc=test,dc=local
Search filter: (sAMAccountName={{username}})
Unique ID Field Mapping: sAMAccountName

I have a Windows server 2016 standard edition. Wiki.js current version 2.5.305, Node.js 18.20.5 PostgreSQL 14.15

**Comment by amal-viswa** [comment]
Hey, i'm having trouble while setting up azure ad authentication for wiki.js , I've followed the wiki.js official documentation for azure ad  , but still not able to login error . can you guys help me resolve it

error -
![Screenshot 2022-08-23 at 18-12-26 Error](https://user-images.githubusercontent.com/90927394/186364687-04aa41a0-4b11-4aee-a7b6-b90c1fd06fe1.png)
